cli/config_pkcs11.go

```diff
@@ -0,0 +1,83 @@
+// +build pkcs11
+
+package cli
+
+import (
+	"flag"
+
+	"github.com/cloudflare/cfssl/config"
+	"github.com/cloudflare/cfssl/signer/universal"
+)
+
+// Config is a type to hold flag values used by cfssl commands.
+type Config struct {
+	Hostname           string
+	CertFile           string
+	CSRFile            string
+	CAFile             string
+	CAKeyFile          string
+	KeyFile            string
+	IntermediatesFile  string
+	CABundleFile       string
+	IntBundleFile      string
+	Address            string
+	Port               int
+	ConfigFile         string
+	CFG                *config.Config
+	Profile            string
+	IsCA               bool
+	IntDir             string
+	Flavor             string
+	Metadata           string
+	Domain             string
+	IP                 string
+	Remote             string
+	Label              string
+	Module, Token, PIN string
+	PKCS11Label        string
```

Comment:
I thought about reusing it at first. However, a command line option can only go to one variable; if we re-use the "-label" flag, it's inconsistent with the other three "pkcs11-" prefixed options. This makes it less obvious that the flag also serves as a PKCS11 label.